### DISCUSSION TITLE
QiitaのMarkdownパーサーの変更をベータ版として公開しました！

### DISCUSSION THREAD

**Discussion by kyntk** [increments/qiita-discussions#139] [アナウンス]
Markdownパーサーの変更をベータ版として公開します！
くわしくは[Qiita Blog](https://blog.qiita.com/replace-markdown-parser-beta/)をご覧ください。

また、新しいMarkdownパーサーにおける注意点について、5日に渡って合計10記事に分けて説明していきます！
様々なMarkdownの記法について解説していきますので、ぜひ[こちらの記事を](https://qiita.com/Qiita/items/e87ebe3138f62b0e8b58)ご覧ください！

**Comment by kyntk** [comment]
Markdownパーサー変更にともなう、見出しについてのいろいろな書き方と注意点をまとめました！
https://qiita.com/degudegu2510/items/a4dbf82d15d80d8bd17e

**Comment by kyntk** [comment]
Markdownパーサー変更にともなう、引用とHTMLブロックについての注意点をまとめました！
https://qiita.com/kyntk/items/4fd09e6fb041edeac2d9

**Comment by uhooi** [comment]
Markdownの詳細な解説記事、非常にわかりやすくて素晴らしいです👏

ただ2点ほど気になりました！

1具体的な変更箇所がわかりづらい
全体的なMarkdown記法についてはわかるのですが、具体的にどこがどう変更されたかがわかりづらいです。
変更箇所の一覧があると嬉しいです。
…と思ったらありました。失礼しました🙇‍♂️
https://help.qiita.com/ja/articles/qiita-markdown
わかりやすいように解説記事にもリンクがあるとよさそうです！

2変更の背景がわからない
エディタの改善のため、と書かれていますが、ぱっと考えただけだとMarkdownパーサーとの関連性がわからないです。
https://blog.qiita.com/replace-markdown-parser-beta/
書けない内容なら大丈夫ですが、可能なら誰でもわかるように背景が書かれていると嬉しいです🙌

**Reply by kskwtnk** [reply]
@uhooi 

ご意見ありがとうございます！
それぞれに回答します！

## 具体的な変更箇所

ヘルプページへのリンク、記事中に入れていたは入れていたのですが埋もれて分かりづらかったためリンクカードの形式に変更しました！

## 変更の背景

エディタ改善の1つとして、プレビューのスクロール位置改善を行う予定です
それに伴い、Markdownパーサーに対して課題が見えてきたためアップデートを実施しました！

現在編集エリアとプレビューのスクロール位置を同期させる開発も進めており、このあたりの話をブログにも追記しようと思います :pray:

**Reply by kskwtnk** [reply]
@uhooi 

ブログへの追記も完了しました！

**Reply by uhooi** [reply]
@xrxoxcxox 迅速なご返信 & ご対応ありがとうございます…！
記事に追記されていることを確認しました ✨ 
変更の背景も理解できました！
確かに今だと編集エリアとプレビューでスクロール位置がずれますよね、、

**Reply by kskwtnk** [reply]
@uhooi 
こちらこそ、より分かりやすくなるためのご意見をくださってありがとうございます！

そうなんですよね……時期はまだ確定できないのですが、位置が揃えば記事を書くのもだいぶやりやすくなると思うので、お待ちください！

**Comment by kyntk** [comment]
Markdownパーサー変更にともなう、コードブロック、コードスパンについてのいろいろな書き方と注意点をまとめました！
https://qiita.com/ohakutsu/items/d9ec356ce9bec4e83ef1

**Comment by kyntk** [comment]
Markdownパーサー変更にともなう、テーブルの書き方をまとめました!
https://qiita.com/aizawa213/items/9c7b1d3a41293e0e3812

**Comment by kyntk** [comment]
Markdownパーサー変更にともなう、リストについての注意点をまとめました！
https://qiita.com/kato-hiroaki/items/b8bcc1e3eb28facf7467

**Comment by kyntk** [comment]
Markdownパーサー変更にともなう、強調と打ち消しについてのいろいろな書き方と注意点をまとめました！
https://qiita.com/fussy113/items/b40f60f6803aa2d5cfbc

**Comment by kyntk** [comment]
Markdownパーサー変更にともなう、脚注についての書き方と注意点をまとめました！
https://qiita.com/phigasui/items/010f4c79ff47bbaaebdd

**Comment by kyntk** [comment]
Markdownパーサー変更にともなう、リンク・オートリンク・Qiita リンクカードの記載方法についてまとめました！
https://qiita.com/mziyut/items/bc00ed7396bd60a717b8

**Comment by kyntk** [comment]
Markdownパーサー変更にともなう、補足説明の書き方と注意点についてまとめました！
https://qiita.com/xrxoxcxox/items/0c91e5319c32434f2c6a
